backend/src/db/migrations/20230826133720_add_role_id_to_users_table.ts

```diff
@@ -0,0 +1,29 @@
+import { type Knex } from 'knex';
+
+const TABLE_NAME = 'users';
+
+const REFERENCE_COLUMN = 'role_id';
+
+const ROLE_MAPPING = {
+  CHATBOT: 1,
+  USER: 2,
+};
+
+function up(knex: Knex): Promise<void> {
+  return knex.schema.alterTable(TABLE_NAME, (table) => {
+    table
+      .integer(REFERENCE_COLUMN)
+      .unsigned()
+      .notNullable()
+      .defaultTo(ROLE_MAPPING.USER);
+    table.foreign(REFERENCE_COLUMN).references('id').inTable('user_roles');
```

Comment:
Create enums and use 'id' and 'user_roles' from there

Reply:
Done!